content/foundation/working-groups/working_groups.toml

```
@@ -0,0 +1,14 @@
+[[working_groups]]
+name = "Website Working Group"
+summary = "Editoral and technical oversight of the main websites and socials"
+description = """
+The website working group is responsible for the editorial and technical oversight of the main Matrix websites and social media channels.
+This includes the main matrix.org website, Event website, and the various social media channels.
```

is this on purpose? should it be `conference website` altogether?
```suggestion
This includes the main matrix.org website, event website, and the various social media channels.
```

---

I went with events as its also "events WG" afaik. But I can just do conference too.

---

I think
- either we generalise it more somehow
- or, we only have the one which is the conference website